Makefile

```diff
@@ -89,7 +89,6 @@ build-windows-amd64:
 
 .PHONY: for-all
 for-all:
-	@$${CMD}
```

Comment:
What does this do?

Reply:
I noticed that most tests were running twice, and believe it was because this line was running the specified command at the root directory. This was in addition to running it wherever a `go.mod` was found.